notebooks/official/automl/sdk_automl_tabular_regression_online_bq.ipynb

```diff
@@ -0,0 +1,915 @@
+{
```

Comment:
<p>Change to:</p><p>BUCKET_NAME = "[your-bucket-name]"&nbsp; # @param {type:"string"}</p><p>BUCKET_URI = f"gs://{BUCKET_NAME}"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/369/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
modified the code to have 2 variables BUCKET_NAME and BUCKET_URI.